matrix/common/aws/redshift_handler.py

```diff
@@ -19,4 +47,13 @@ def run_query(self, query):
         conn = pg.connect(self.database_uri)
         cursor = conn.cursor()
         cursor.execute(query)
+        conn.commit()
+        conn.close()
+
+    def transaction(self, queries: typing.List[str]):
```

Comment:
can we just replace run_query with transaction in the query_runnner (invoked with a list of one query)? This way we can delete one of these functions

Reply:
Yup, didn't want to introduce any unnecessary regressions but I'll go ahead and make that change